apps/design/frontend/src/contests_screen.tsx

```diff
@@ -45,16 +47,26 @@ import { ElectionNavScreen } from './nav_screen';
 import { ElectionIdParams, electionParamRoutes, routes } from './routes';
 import { TabPanel, TabBar } from './tabs';
 import { getElection, updateElection } from './api';
-import { generateId, replaceAtIndex } from './utils';
+import { generateId, reorderElement, replaceAtIndex } from './utils';
+
+const ReorderableTr = styled.tr<{ isReordering: boolean }>`
+  &:hover {
+    background-color: ${(p) => p.isReordering && p.theme.colors.containerLow};
+  }
+`;
 
 const FILTER_ALL = 'all';
 const FILTER_NONPARTISAN = 'nonpartisan';
 
 function ContestsTab(): JSX.Element | null {
   const { electionId } = useParams<ElectionIdParams>();
   const getElectionQuery = getElection.useQuery(electionId);
+  const updateElectionMutation = updateElection.useMutation();
   const [filterDistrictId, setFilterDistrictId] = useState(FILTER_ALL);
   const [filterPartyId, setFilterPartyId] = useState(FILTER_ALL);
+  const [reorderedContests, setReorderedContests] = useState<
+    Contests | undefined
+  >();
```

Comment:
Unnecessary `| undefined`.
```suggestion
  const [reorderedContests, setReorderedContests] = useState<Contests>();
```